bin/run-fisher.sh

```diff
@@ -14,4 +16,5 @@ python -u DeepSpeech.py \
   --epoch=20 \
   --display_step 1 \
   --validation_step 5 \
+  --checkpoint_dir "$XDG_DATA_HOME/$(basename ${0%.*})" \
```

Comment:
…and at that point, you might as well hardcode the importer name in each script :P